segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ReadOperation.java

```diff
@@ -90,7 +90,7 @@
                                         return readData(txn);
                                     }, chunkedSegmentStorage.getExecutor())
                                     .exceptionally(ex -> {
-                                        log.debug("{} read - started op={}, segment={}, offset={}, bytesRead={}.",
+                                        log.debug("{} read - exception op={}, segment={}, offset={}, bytesRead={}.",
```

Comment:
If this is an exception, why it is in `debug` level? In general, we want to keep track of relevant exceptions in production systems, which requires this to be at least `info` level, if not `warn` or `error`. As SLTS is a new piece of code, let's keep high the log level of exceptions so we can debug problems quickly without needed to repeat experiments with `debug` level logs.

Reply:
We do throw this exception which the caller will log appropriately.
This is just additional debug information that exception does not contain.